src/main/loadfragment2.c

```diff
@@ -72,7 +84,8 @@ void DoRelocation(void* allocatedRamAddr, OverlayRelocationSection* ovl, uintptr
                 // Check address is valid for relocation
                 if ((*relocDataP & 0x0F000000) == 0) {
                     *relocDataP = *relocDataP - vramStart + allocu32;
-                } else if (gLoad2LogSeverity >= 3) {
+                } else if (gOverlayLogSeverity >= 3) {
+                    (void)"セグメントポインタ32です %08x\n";
```

Comment:
A comment with a rough translation would be nice

Reply:
I think I got them all.